tsconfig.json

```diff
@@ -1,18 +1,26 @@
 {
   "compilerOptions": {
-    "target": "esnext",
+    "target": "es2017",
     "module": "esnext",
     "moduleResolution": "node",
-    "lib": ["ESNext", "dom"],
+    "declaration": true,
+		"lib": ["es5", "es2015", "es2016", "es2017", "es2018", "es2019", "es2020", "esnext","DOM"],
     "allowSyntheticDefaultImports": true,
     "strict": true,
+    "resolveJsonModule": true,
     "noImplicitThis": true,
     "alwaysStrict": true,
-    "types": ["node"],
+    "types": [
+      "node"
+    ],
```

Comment:
Is this just a random auto-formatting thing that happened? Maybe this repo needs to specify a Prettier config so we can avoid having our personal config differences cause unnecessary formatting diffs?